index.html

```diff
@@ -21,6 +22,10 @@
                 </div>
                 <nav class="toolbar">
                     <ul>
+                        <li><a href="#" class="btn-close"><i class="fa fa-close"></i></a></li>
+                        <li><a class="btn-menu-item1" href="#portfolio">Portfolio</a></li>
+                        <li><a class="btn-menu-item2" href="#about">About</a></li>
+                        <li><a class="btn-menu-item3" href="#contact">Contact</a></li>
                         <li class="collapse-button"><i class="fa fa-list"></i></li>
                         <li class="menu">
                             <span>
```

Comment:
- Hey Team, please, see these lines of [codes in your HTML file](https://github.com/Elhambasir/MyPortfolio/blob/06c6ae51624a400936a3363ecac05a920b171aac/index.html#L30-L38), there's a duplicate in your mobile menu links, please, kindly remove the duplicates to make your design close to the Figma design as much as you can.


![image](https://user-images.githubusercontent.com/59397159/223355234-368d1de0-ad05-49b5-bbfc-9e96e9da3da8.png)